icssc/peterportal-client
# Sync local roadmap with cloud

## Description
* prompt the user to load their local roadmap if it is newer than the one saved to their account in mongo
* added a timestamp to the roadmap object for the sake of determining which is newer
* as of right now, after loading the local roadmap, you have to click save to save it to your account in the db
  * refreshing the page will bring up the prompt again
  * maybe could add a toast or some other message letting the user know to save if they want to keep these changes
* also, it takes a second to load after clicking yes, may need to add a loading indicator or something

## Screenshots

![image](https://github.com/icssc/peterportal-client/assets/8922227/25742169-8eca-4cfe-8c7f-834e4b230f0c)

## Steps to verify/test this change:

- [ ] Verify changes work as expected on staging instance
  1. Log out
  2. make changes to schedule and save
  3. log back in, prompt should now appear
  4. if you save, roadmap will be saved both to db and localstorage, prompt should no longer appear if you refresh now

## Final Checks:

- [ ] Verify successful deployment

(optional)


## Issues



Closes #313
